.github/workflows/codeql.yml

```diff
@@ -0,0 +1,69 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ "master" ]
+
+jobs:
+  analyze:
+    name: Analyze
```

Comment:
Usually, we don't specify names for workflows. If it is not defined, `analyze` will be used.

Reply:
Although I think it would be better to have a human-readable name, I removed it.